### PR TITLE
Add miRNAture

### PR DESCRIPTION
**Pull request by jfallmann**
Describe your pull request here

----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

For members of the Bioconda project, the following command is also available:

<table>
  <tr>
    <td><code>@BiocondaBot please merge</code></td>
    <td>Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts.</td>
  </tr>
</table>

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

**Comment by jfallmann** [comment]
@BiocondaBot please add label

File: recipes/mirnature/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+  - perl-file-copy-recursive=0.45
+  - perl-file-share=0.25
+  - perl-file-sharedir=1.116
+  - perl-statistics-r=0.34=pl526r351_3
+  - r-base=3.5.1
+  - r-dplyr=0.8.5
+  - viennarna=2.4.17
+  
+  run:
+  - blast=2.9.0
```
Comment (eggzilla) [inline recipes/mirnature/meta.yaml:40]: You have a lot of pins to an exact version. I would recommend relaxinf the version constraints, and
allow newer versions. Specifically for blast you should consider using 2.10.1 if possible

File: recipes/mirnature/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+  - r-base=3.5.1
+  - r-dplyr=0.8.5
+  - rmblast=2.9.0
+  - viennarna=2.4.17
+
+test:
+  imports:
+    - RNA
+  commands:
+    - miRNAture -h | grep man
```
Comment (eggzilla) [inline recipes/mirnature/meta.yaml:70]: I suggest removing the dependency on grep here. I know that testing with help output is very widespread now, but
if you could use a minimal runtime example to test that would be awesome

File: recipes/mirnature/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+  - perl-file-share=0.25
+  - perl-file-sharedir=1.116
+  - perl-statistics-r=0.34=pl526r351_3
+  - r-base=3.5.1
+  - r-dplyr=0.8.5
+  - viennarna=2.4.17
+  
+  run:
+  - blast=2.9.0
+  - blast-legacy=2.2.26
```
Comment (eggzilla) [inline recipes/mirnature/meta.yaml:41]: Do you really need blast and blast-legacy?

**Comment by eggzilla** [comment]
@jfallmann , I added some recommendations for your package. Please get back to me once you reviewed them :-)

File: recipes/mirnature/meta.yaml (hunk 1 above)
Comment (jfallmann) [reply recipes/mirnature/meta.yaml:40]: Blast 2.9 is required as it is the tested version, 2.10 may change output, for this release of the software only 2.9 tests are available

File: recipes/mirnature/meta.yaml (hunk 2 above)
Comment (jfallmann) [reply recipes/mirnature/meta.yaml:70]: more exhaustive tests are run by Build.PL in the build process

File: recipes/mirnature/meta.yaml (hunk 3 above)
Comment (jfallmann) [reply recipes/mirnature/meta.yaml:41]: Author says yes, specific blast strategies rely on that

**Comment by eggzilla** [comment]
Ok, thanks for commenting, i will approve

**Review by eggzilla — APPROVED** [review]

**Comment by jfallmann** [comment]
@eggzilla Thanks!

**Comment by jfallmann** [comment]
@BiocondaBot please merge

**Pull request merged**